.gitlab-ci.yml

```
@@ -94,7 +94,8 @@ changelog:
     - git remote add github-${CI_JOB_ID} https://${GITHUB_USER_NAME}:${GITHUB_BOT_TOKEN_REPO_FULL}@github.com/${GITHUB_REPO_URL} || true # Ignore already existing remote
     - gh repo set-default https://${GITHUB_USER}:${GITHUB_BOT_TOKEN_REPO_FULL}@github.com/${GITHUB_REPO_URL}
     # deal with the bundled changelog
-    - RELEASE_PLEASE_PR=$(gh pr list --author "${GITHUB_USER_NAME}" --head "release-please--branches--${CI_COMMIT_REF_NAME}" | grep -v component | awk '{print $1}')
+    # since the CHANGELOG is for the entire repo we pick the first release PR to update here for now, the components get their own log updated further down
+    - RELEASE_PLEASE_PR=$(gh pr list --author "${GITHUB_USER_NAME}" --head "release-please--branches--${CI_COMMIT_REF_NAME}" | grep component | head -n1 | awk '{print $1}')
```

This is the main PR, the non-component one. In our case, it's already been merged, so it isn't showed up, so we can skip the PR update:
```suggestion
    - RELEASE_PLEASE_PR=$(gh pr list --author "${GITHUB_USER_NAME}" --head "release-please--branches--${CI_COMMIT_REF_NAME}" | grep component | head -n1 | awk '{print $1}' || echo "notfound")
    - if [[ "${RELEASE_PLEASE_PR}" != "notfound" ]]; then 
        gh pr checkout --force $RELEASE_PLEASE_PR;
        wget ....
      else
        echo "the PR for the bundle component is already merged - skipping"
      fi
```